data/utms.py

```diff
@@ -647,25 +697,37 @@ def make_result(status: str) -> RunResult:
             # instruction, so there's actually nothing to do.
             pass
         case ']':
-          # Unconditional jump. Wikipedia mentions that we could check the
-          # condition here too to avoid one step, but treating it as
-          # unconditional makes the logic simpler and avoids more bugs (since
-          # we have complicated the logic somewhat).
-          if new_instruction:
-            if unmatched_brackets:
-              # We just generated a closing bracket, and there are some
-              # opening bracket hanging, so we match these two.
-              open_pos = unmatched_brackets.pop()
-              jumps[program_index] = open_pos
-              # Jump to there. -1 to ensure the next instruction is the
-              # opening bracket.
-              program_index = open_pos - 1
+            # Unconditional jump. Wikipedia mentions that we could check the
```

Comment:
There's an additional level of indentation. I'm surprised Python didn't shout here.

Reply:
Should be fixed now, also found a smaller indentation problem on a later line and fixed that too! Thanks for noticing.